modules/apps/headless/headless-object/headless-object-impl/src/main/java/com/liferay/headless/object/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java

```diff
@@ -421,7 +421,7 @@ private Long _getParentObjectEntryFolderId(
 		return persistedObjectEntryFolder.getObjectEntryFolderId();
 	}
 
-	private ObjectEntryFolder _patchObjectEntryFolder(
+	private ObjectEntryFolder _updateObjectEntryFolder(
```

Comment:
One question reviewing the code, and I know is outside this changes.
```
ServiceContextBuilder.create(
					persistedObjectEntryFolder.getGroupId(),						persistedObjectEntryFolder.getGroupId(),
					contextHttpServletRequest, null						contextHttpServletRequest, null
```
maybe we were deleting the assetTags/categories, and the viewPermissions?

maybe we should attach them to the new serviceContext

Reply:
@AliciaGarciaGarcia, let's do this in another task

Reply:
<img src="https://media3.giphy.com/media/v1.Y2lkPWJkM2VhNTdlcTAxcHAzYmphanY5bGZwcnZsYXdsczEya291bWNsam55aXNodWFoaCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/V3KPDF1wg5tkcTDqk5/giphy.gif"/>